content/200-concepts/100-components/02-prisma-client/190-deployment.mdx

```diff
@@ -74,18 +74,43 @@ postgresql://USER:PASSWORD@HOST:PORT/DATABASE?connection_limit=1
 mysql://USER:PASSWORD@HOST:PORT/DATABASE?connection_limit=1
 ```
 
+### Usage with PgBouncer
+
+Typically PostgreSQL supports only a certain amount of concurrent connections and this limit can be reached quite fast when the service usage goes up – especially in serverless environments.
+
+A connection pooler like PgBouncer prevents your application from exhausing the database's connection limit.
+
+PgBouncer holds a connection pool to the database and proxies incoming client connections by sitting between Prisma Client and the database. This reduces the number of processes a database has to handle at any given time. PgBouncer passes on a limited number of connections to the database and queues additional connections for delivery when space becomes available.
+
+To use Prisma Client with PgBouncer from a serverless function, add the `?pgbouncer=true` flag to the PostgreSQL connection string:
+
+```
+postgresql://USER:PASSWORD@HOST:PORT/DATABASE?pgbouncer=true
+```
+
+Additionally, for Prisma Client to work reliably, PgBouncer must run in [**Transaction mode**](https://www.pgbouncer.org/features.html).
+
+_Transaction mode_ offers a connection for every transaction. With this mode in addition to the , the Prisma query engine is able to operate normally.
+
+<details>
```

Comment:
Seems like the content in here doesn't render properly as Markdown right now:

![image](https://user-images.githubusercontent.com/4058327/105872156-5eb03100-5ffa-11eb-8293-75bc72d554d3.png)

Maybe try to add new lines before and after the bullet list? If that doesn't work, we should summon Nilu to fix the rendering.

Reply:
Yep, this was a whitespace problem. Adding one before and after the bullet list fixed it:
<img width="717" alt="Screen Shot 2021-01-26 at 18 53 58" src="https://user-images.githubusercontent.com/1992255/105884072-da64aa80-6007-11eb-9318-f3a0f98091c2.png">